packages/lib/src/artifacts/ZWeb3.ts

```diff
@@ -0,0 +1,177 @@
+import { promisify } from 'util';
+import sleep from '../helpers/sleep';
+
+// Reference: see https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#list-of-chain-ids
+const NETWORKS = {
+  1:  'mainnet',
+  2:  'morden',
+  3:  'ropsten',
+  4:  'rinkeby',
+  42: 'kovan'
+};
+
+// TS-TODO: Type Web3.
+// TS-TODO: Review what could be private in this class.
+export default class ZWeb3 {
```

Comment:
I don't remember if we discussed this, but personally, I'm not comfortable with having classes full of static methods when we could have objects instead. I know you proposed to discuss this on https://github.com/zeppelinos/zos/issues/434 's phase 6, but I also consider this is not an implementation (javascript/typescript) concern but rather a design issue.

Reply:
Yeah, I also like to stay away from "static classes" or singleton-ish patterns, but it wasn't my task to change the existing design. I found an object being used "statically" and I figured that a "static class" would serve as a first off translation. It works. I would re-address this discussion at a later phase when we focus on design.

Reply:
IMO, if we turn plain objects into classes we are changing the existing design, actually, I've been having many conflicts due to this. Can we stick to the kind of object that was decided during the migration please?